proteus/mprans/BoundaryConditions.py

```diff
@@ -1422,37 +1419,39 @@ def getFluidVelocityLocalCoords(self, xi, element, rank):
             u = v = w = None
         return u, v, w
 
-    def setYplusNormalDirection(self, x, t, relax=1.0):
+    def setYplusNormalDirection(self, x, t,n, relax=1.0):
         """
         Return the point at y+ distance in normal
         direction from the boundary.
         """
         # near wall point
-        nP = (relax * self.Y * (self.nV)) + x
+        nP = (relax * self.Y * n) + x
```

Comment:
Since n is the outward pointing normal on the fluid domain boundary, shouldn't  it be (-n)?